src/containers/CreateEventForm.jsx

```diff
@@ -46,8 +46,14 @@ const FormikCreateEventForm = withFormik({
       attendanceCode: values.attendanceCode,
       description: values.description
     }
-    props.postEvent(event);
-    resetForm();
+    props.postEvent(event)
+    .then((resp) => {
+      resetForm();
+    })
+    .catch((error) => {
```

Comment:
I don't think we need to log the error since we already notified the user in the UI